### DISCUSSION TITLE
Lengthy column name [May 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5732] [Discord Threads]
> 🙋‍♂️ **ilyak**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Can you provide more details about your setup? You can copy project info & paste it here.  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### https://github.com/nocodb/nocodb/issues/5438  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### created an issue. please check if this is what you intended  
#  
> 🙋‍♂️ **ilyak**  
#### Apologies, missed it. Do I still need to do anything re that? Or the issue is clear?  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### no. we expect this fix to be available in a release or two  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1093871306677686375)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
